### DISCUSSION TITLE
Creative Commons License Support

### DISCUSSION THREAD

**Discussion by wizardbyron** [jpanther/congo#56] [Ideas]
I've added [Creative Commons License](https://creativecommons.org/) statement on my blog.

The implementation is create a partial page `license.html` in `single.html` at following position:

```html
    <footer class="pt-8">
      {{ partial "author.html" . }}
      {{ partial "license.html" . }} <<- HERE
      {{ partial "sharing-links.html" . }}
      {{ partial "article-pagination.html" . }}
    </footer>
```

I'm not sure which position is better? Before or after `sharing-links.html`?

And I'm thinking about add  the following options  with i18n support:

1. false(default)
2. BY
3. BY-SA
4. BY-ND
5. BY-NC
6. BY-NC-ND
7. BY-NC-SA

The good news is that the CC site provides the i18n versions, so I can handle i18n by myself.

Would you like add the CC statement as a new feature?

**Comment by jpanther** [comment]
Thanks for the suggestion. I'm thinking this may be adding unnecessary complication. Do you have a particular need to have a different licence for each article? If you are licensing your content using Creative Commons why not simply put the licence details in the `copyright` string and display it in the site footer?

**Reply by wizardbyron** [reply]
Good idea, forget this one.
